other-docs/guides/upgrading/v13.md

```diff
@@ -0,0 +1,66 @@
+---
+order: 13
+---
+# Upgrading to v13
+
+_If you are migrating from WordPress to Altis, check out the [migrating guide here](../migrating-from-wordpress.md) first._
+
+To upgrade to Altis v13, edit your `composer.json` and change the version constraint for `altis/altis` and any local environment modules to `^13.0.0`. Note that the new default and recommended version of PHP for Altis v13 is now 8.0. Make sure you have tested your custom code and any additional plugins for compatibility with [PHP version 8.0](#php-80).
+
+```json
+{
+	"require": {
+		"altis/altis": "^13.0.0"
+	},
+	"require-dev": {
+		"altis/local-server": "^13.0.0"
+	},
+	"config": {
+		"platform": {
+			"php": "8.0"
+		}
+	}
+}
+```
+
+Once you have made these changes run `composer update` and then run the `wp altis migrate` command:
+
+```sh
+# For cloud environments
+wp altis migrate
+
+# For local server
+composer server cli -- altis migrate
+```
+
+## Breaking Changes
+
+### PHP 8.1 ###
+
+Altis v13 fully supports PHP 8.1 in both local and cloud environments. Support for versions of PHP earlier than 8.0 has been removed in Altis v13. There are a number of [backward incompatible changes](https://www.php.net/manual/en/migration81.incompatible.php) in PHP 8.1 which should be addressed.
+
+Refer to our [PHP Version Guide](docs://guides/updating-php/) for up-to-date compatibility, testing and upgrading information.
+
+### PHP 7.4 ###
+
+The release of Altis v13 introduces the removal of support for PHP 7.4, follow the [PHP Version Guide](docs://guides/updating-php/) for information on how to upgrade to PHP 8.0 or PHP 8.1.
+
+
+## Headline Features
+
+### WordPress 6.0.1 ###
+
+Wordpress 6.0.1 brings with it a lot of performance improvements to the block editor, taxonomies and the cache API amongsts many others.
```

Comment:
WordPress with a capital P please. Also I think this section should be about WordPress 6.0 as that's the new major release in Altis v13, 6.0.1 and 6.0.2 are bugfixes/maintenance/security patches.

Reply:
Thanks for pointing this out, it has been corrected now.